pnpm-lock.yaml

```diff
@@ -46,7 +46,7 @@ importers:
         version: 0.6.2
       ai:
         specifier: 2.2.12
-        version: link:../../packages/core
+        version: 2.2.12(react@18.2.0)(solid-js@1.7.7)(svelte@4.0.1)(vue@3.3.4)
```

Comment:
We might need to revert this change.

Reply:
Yeah, don't know why it's even there to be honest. Maybe I have a different set package manager setup. I'll try to figure it up.

Reply:
@aberezkin if you can fix the merge conflic tin `use-chat` and re-run `pnpm i` from the root directory this should be resolved and we can merge this PR.